drivers/gyro/adxrs290/iio_adxrs290.h

```diff
@@ -42,116 +42,7 @@
 
 #include "iio_types.h"
 
-ssize_t get_adxrs290_iio_ch_raw(void *device, char *buf, size_t len,
```

Comment:
Were the function headers removed on purpose? I see the non-static implementations are still available in the *.c file.

Reply:
They are in the .c I will make them static.

Reply:
I suggest making a separate commit for this change.

Reply:
Because it is unrelated to the commit title/description

Reply:
Adding static is related to this commit. If removing from .h implies make them static.
Also they are moved because struct iio_adxrs290_descriptor is moved so I think to split this neither would make sense

Reply:
Ack